roles/kubernetes-apps/external_cloud_controller/openstack/templates/external-openstack-cloud-config.j2

```diff
@@ -36,6 +36,12 @@ subnet-id={{ external_openstack_lbaas_subnet_id }}
 {% if external_openstack_lbaas_floating_network_id is defined %}
 floating-network-id={{ external_openstack_lbaas_floating_network_id }}
 {% endif %}
-{% if external_openstack_lbaas_flaoting_subnet_id is defined %}
+{% if external_openstack_lbaas_floating_subnet_id is defined %}
 floating-subnet-id={{ external_openstack_lbaas_floating_subnet_id }}
 {% endif %}
+{% if external_openstack_lbaas_manage_security_groups is defined %}
+manage-security-groups={{ external_openstack_lbaas_manage_security_groups }}
+{% endif %}
+{% if external_openstack_lbaas_internal_lb is defined %}
+internal-lb={{ external_openstack_internal_lb }}
```

Comment:
You can also specify it through annotations: https://github.com/kubernetes/cloud-provider-openstack/blob/master/examples/loadbalancers/internal-http-nginx.yaml#L27
But it's good to have it here also.

Reply:
Good to know, still good to have an option to set it as default though.